src/assets/scripts/client/InputController.js

```diff
@@ -713,6 +714,16 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     */
+    _disableRightClickMenu() {
```

Comment:
Just an example, this is what happens when you right click currently (screengrab from #576 deployment) 
![image](https://user-images.githubusercontent.com/29643177/28494477-655eae22-6ef5-11e7-8de7-c64662f98858.png)